common/src/main/java/org/apache/nemo/common/punctuation/Latencymark.java

```diff
@@ -44,26 +44,33 @@ public Latencymark(final String taskId, final long timestamp) {
   }
 
   /**
-   * @return the latencymark timestamp
+   * @return the latencymark timestamp.
    */
   public long getCreatedTimestamp() {
     return createdTimestamp;
   }
 
   /**
-   * @return the task id where it is created
+   * @return the task id where it is created.
    */
   public String getCreatedTaskId() {
     return createdTaskId;
   }
 
   /**
-   * @return the task id of previous task
+   * @return the task id of previous task.
```

Comment:
Could you provide more information in this comments on what `previousTaskId` is for?
According to the comments of `LatencyMark` class, this class is for recording task ID and time when the task is created. Does this class assume that multiple tasks deal with a single source vertex and exchange information (since there are notions of 'task' and 'previous task') ?